### DISCUSSION TITLE
Выпадение оружия при вылете

### DISCUSSION THREAD

**Discussion by Xacan** [octothorp-team/public-dobrograd#860] [Предложения]
Когда мы держим предметы в слоте "Руки" в инвентаре и выходим с сервера либо вылетаем, предметы выпадут на землю. Но если держать оружие не в слоте, а просто в руках и пропасть с сервера, пушка исчезнет вместе с тобой. Предлагаю чтоб она выпадала на землю, чтоб друзья, если таковые есть, смогли засейвить ее

**Comment by PetrBronzo** [comment]
Можно лучше: У многих игроков плохой интернет и на сервере часто орудуют челики DDos. Поэтому лучшим вариантом является авто-компенсация, возврат денег за вещи в руках в размере 50% от их стоимости (за все вещи кроме оружия, так как сталь убрали из магазина)
